packages/api/tests/models/rules/test-rules-model.js

```diff
@@ -196,6 +196,19 @@ test.serial('Creating an invalid rule does not create workflow triggers', async
   }
 });
 
+test('Creating a rule with a createdAt and updatedAt date assigns the correct values to createdAt and provides a new updatedAt', async (t) => {
+  const newRule = fakeRuleFactoryV2();
+  const currentDate = new Date('1/2/2030').getTime();
+  newRule.workflow = workflow;
+
+  delete newRule.collection;
+  delete newRule.provider;
```

Comment:
I don't know if this is necessary in a test for just the Dynamo `create`?

Reply:
Sorry, do you mind elaborating? What do you think is unnecessary? Deleting collection and provider?

Reply:
Deleting the collection/provider is something you're doing in the tests that write to RDS because in RDS a collection/provider is required but you don't want to always pre-create those records in RDS. In Dynamo, collection/provider aren't required per se, so we don't need to delete those attributes of the rule before trying to save. Unless I'm missing something

Reply:
Oh I see. Yes you are totally right, it's not required in the dynamo schema.